### PR TITLE
Upgraded deepl-cli code

### PR DESCRIPTION
**Pull request by Monsterovich**
I wanted to make one change at first, but decided to put everything into one PR. This PR improves the following:
- Added support for skeleton display of translated text. Fixes https://github.com/eggplants/deepl-cli/issues/253.
- Fixed a bug using the wrong button locator when `use_dom_submit = True`.
- The code now also closes the Chrome extension popup that possibly can prevent clicking on language buttons (also for `use_dom_submit = True`).
- Update: also changed the max chars limit from 3000 to 1500.

### CODE REVIEW

**Comment by Monsterovich** [comment]
ping @eggplants

File: deepl/deepl.py [hunk 1]
```diff
@@ -133,15 +133,39 @@ async def __translate(self, script: str) -> str:
...
+                except PlaywrightError:
+                    pass
+
+                # try to close the extension banner
+                try:
+                    await page.evaluate(
+                        """
+                         document.querySelector('div[data-testid="chrome-extension-toast"]').querySelector('button').click()
+                    """
+                    )
```
Comment (eggplants) [inline deepl/deepl.py:152]:
```suggestion
                    await page.evaluate(
                        """
                        document.querySelector(
                            'div[data-testid="chrome-extension-toast"]',
                        ).querySelector('button').click()
                        """,
                    )
```

File: deepl/deepl.py [hunk 2]
```diff
@@ -153,20 +177,59 @@ async def __translate(self, script: str) -> str:
...
                 raise DeepLCLIPageLoadError(msg) from e
 
+            # Get the number of lines in the translated text field
+            try:
+                line_count = await page.evaluate(
+                    """
+                     document.querySelector(
+                        'd-textarea[aria-labelledby=translation-target-heading]').children[0].children.length
+                """
+                )
```
Comment (eggplants) [inline deepl/deepl.py:187]:
```suggestion
                line_count = await page.evaluate(
                    """
                    document.querySelector(
                        'd-textarea[aria-labelledby=translation-target-heading]',
                    ).children[0].children.length
                    """,
                )
```

File: deepl/deepl.py [hunk 3]
```diff
@@ -153,20 +177,59 @@ async def __translate(self, script: str) -> str:
...
+                    translated_text = await page.evaluate(
+                        f"""
+                         document.querySelector(
+                            'd-textarea[aria-labelledby=translation-target-heading]').children[0].children[{line}].innerText
+                    """
+                    )
+                    translated_lines.append(translated_text)
+                except PlaywrightError as e:
+                    msg = f"Unable get translated text for line {line}"
+                    raise DeepLCLIPageLoadError(msg) from e
```
Comment (eggplants) [inline deepl/deepl.py:219]:
```suggestion
            for line_index in range(line_count):
                try:
                    await page.wait_for_function(
                        f"""
                        () => {
                            const t = document.querySelector(
                                'd-textarea[aria-labelledby=translation-target-heading]',
                            )?.children[0]?.children[{line_index}]?.innerText ?? '';
                            t.length > 0 && !t.startsWith('[...]')
                        """,
                    )
                except PlaywrightError as e:
                    msg = f"Time limit exceeded for line {line_index}. ({self.timeout} ms)"
                    raise DeepLCLIPageLoadError(msg) from e

                try:
                    translated_text = await page.evaluate(
                        f"""
                        document.querySelector(
                            'd-textarea[aria-labelledby=translation-target-heading]'
                        ).children[0].children[{line_index}].innerText
                        """,
                    )
                    translated_lines.append(translated_text)
                except PlaywrightError as e:
                    msg = f"Unable get translated text for line {line_index}"
                    raise DeepLCLIPageLoadError(msg) from e
```

**Comment by Monsterovich** [comment]
@eggplants Now the code doesn't run:

```
Traceback (most recent call last):
  File "/home/monsterovich/Scripts/deepl_translate_clipboard.py", line 7, in <module>
    from deepl import deepl
  File "/home/monsterovich/.local/lib/python3.10/site-packages/deepl/__init__.py", line 3, in <module>
    from .deepl import DeepLCLI, DeepLCLIError
  File "/home/monsterovich/.local/lib/python3.10/site-packages/deepl/deepl.py", line 203
    const t )
    ^^^^^^^
SyntaxError: f-string: invalid syntax. Perhaps you forgot a comma?
```

**Comment by eggplants** [comment]
Thank you for your contribution,

This codebase is corrupt for a while due to deepl's update and lack of maintenance for deepl-cli. Inappropriate fixtures will be fixed and be published to PyPI with some fixture to make translation successfully soon.

**Comment by Monsterovich** [comment]
> Thank you for your contribution,
> 
> This codebase is corrupt for a while due to deepl's update and lack of maintenance for deepl-cli. Inappropriate fixtures will be fixed and be published to PyPI with some fixture to make translation successfully soon.

Eh, just quickly merge this -> https://github.com/eggplants/deepl-cli/pull/268

**Pull request merged**